EnglishFilter/sections/general_extensions.txt

```diff
@@ -229,6 +229,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75284
+curimovie.link#%#//scriptlet("prevent-window-open")
```

Comment:
Not required ?blocks social buttons

Reply:
@ghajini This is required for AdGuard Apps. Otherwise, pop-ups still stays

Reply:
ok i have no adguard apps so can't check

Reply:
It's Okay I could reproduce it. I will fix it soon.